rust/ql/lib/codeql/rust/frameworks/Reqwest.qll

```diff
@@ -0,0 +1,16 @@
+/**
+ * Provides modeling for the `reqwest` library.
+ */
+
+private import rust
+private import codeql.rust.Concepts
+
+/**
+ * A call to `reqwest::get` or `reqwest::blocking::get`.
+ */
+private class ReqwestGet extends RemoteSource::Range {
+  ReqwestGet() {
```

Comment:
I think this QLL file is fine, but I would hope we can delete it soon and replace it with "models as data".

Reply:
100% agree, and the same applies for pretty much all of the modelling work I'm doing at the moment.